docs/en/enterprise-edition/content-collections/governance/custom-build-policies/visual-editor.adoc

```diff
@@ -39,7 +39,7 @@ The *IaC Policy Code Editor* is displayed by default in the *Create query* step
 [#configure-iac]
 === Configure IaC Custom Build Policies
 
-Limitation: The Visual Editor supports policy creation for Prisma Cloud's supported cloud service providers. For unsupported providers, use the Code Editor.
+Limitations: The Visual Editor supports policy creation for cloud service providers supported by Prisma Cloud. For unsupported providers, use the Code Editor. Additionally, the Resource Type and Attribute fields are selection only and not free text. For more advanced policies that require custom attributes, leverage the Code Editor. For example, tagging rules often require attributes like `tags.<tag_key>` which cannot be accomplished with the Visual Editor. Finally, the Visual Editor does not support connected resources, use the Code Editor for these policies.
```

Comment:
**Limitations**
-  The Visual Editor supports policy creation for cloud service providers supported by Prisma Cloud.  For unsupported providers, use the Code Editor
- The 'Resource Type' and 'Attribute' fields are selection-based, not free text
-  For more advanced policies requiring custom attributes, such as tagging rules with `tags.<tag_key>`, use the Code Editor
- The Visual Editor does not support connected resources; use the Code Editor for these policies